meilisearch/tests/index/test_index_search_meilisearch.py

```diff
@@ -138,6 +138,20 @@ def test_custom_search_params_with_facet_filters(self):
         assert 'facetsDistribution' not in response
         assert 'exhaustiveFacetsCount' not in response
 
+    def test_custom_search_params_with_multiple_facet_filters(self):
+        update = self.index.update_attributes_for_faceting(['genre'])
+        self.index.wait_for_pending_update(update['updateId'])
+        response = self.index.search(
+            'world',
+            {
+                'facetFilters': ['genre:action', ['genre:action', 'genre:action']]
```

Comment:
Is this just to check if syntax is OK and that the request doesn't throw an error? otherwise, I don't really understand the point on using 3 times the same attribute and value 😄

Reply:
It's just to check that the array value can pass. Not to check that MeiliSearch does correctly its job.
The behavior of MeiliSearch with this weird array should not be the point of the SDK test, so we don't care if there is 3 times the same value ;)
(Plus we do not provide a better dataset for our tests with multiples facets to check that MS works correctly and it would imply a lot of other tests if we really want to check the MS behavior with the facets)